agent/engine/docker_image_manager.go

```diff
@@ -84,6 +84,12 @@ func (imageManager *dockerImageManager) AddAllImageStates(imageStates []*image.I
 	}
 }
 
+func (imageManager *dockerImageManager) GetCountImageStates() int {
```

Comment:
I'd thought about that, two reasons why:
- Returning the underlying data leaks the abstraction
- I don't think we need that functionality anywhere yet